.kokoro/stage.sh

```diff
@@ -23,17 +23,30 @@ source "${dir}"/common.sh
 
 pushd "${dir}"/../
 
+# Set up environment variables used in release.
 MAVEN_SETTINGS_FILE=$(realpath .)/settings.xml
-
+GRADLE_SETTING_FILE=$(realpath .)/app-gradle-plugin/gradle.properties
 setup_environment_secrets
 create_settings_xml_file "${MAVEN_SETTINGS_FILE}"
+create_gradle_properties_file "${GRADLE_SETTING_FILE}"
 
 # Use GCP Maven Mirror
 mkdir -p "${HOME}"/.m2
 cp settings.xml "${HOME}"/.m2
 
 gcloud components install app-engine-java --quiet
 
+# Release app-gradle-plugin
```

Comment:
I change the release order to release gradle plugin first, so that if something wrong in gradle command, we can change the script and rerun it.